### PR TITLE
slo: dedicated slo resource in terraform provider

### PR DESCRIPTION
**Pull request by dwalasek**
slo: dedicated slo resource in terraform provider

### CODE REVIEW

File: signalfx/resource_signalfx_detector.go [hunk 1]
```diff
@@ -20,6 +20,62 @@ const (
...
+		},
+		"parameterized_body": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Custom notification message body when an alert is triggered. See https://developers.signalfx.com/v2/reference#detector-model for more info",
+		},
+		"parameterized_subject": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Custom notification message subject when an alert is triggered. See https://d    evelopers.signalfx.com/v2/reference#detector-model for more info",
```
Comment (ahoubowicz) [inline signalfx/resource_signalfx_detector.go]: looks like there are some spaces inside the url

Comment (dwalasek) [reply signalfx/resource_signalfx_detector.go]: Good catch. Fixed

**Review by ahoubowicz — APPROVED** [review]

**Review by ahoubowicz — APPROVED** [review]

**Review by benkeith-splunk — APPROVED** [review]
Looks good.

**Comment by benkeith-splunk** [comment]
@dwalasek you can upgrade the Go version in go.mod to 1.20 and update the CI to use that.

**Comment by dwalasek** [comment]
@benkeith-splunk Thx for review, I've upgraded go version - could you please approve again?

**Comment by benkeith-splunk** [comment]
The CI thing isn't working, see https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go#using-a-specific-go-version and try upgrading the setup-go action to v4.

**Comment by dwalasek** [comment]
@benkeith-splunk I updated the setup-go action to version 4, but the error message is not really meaningful to me. Sorry to bother you, but could you restart the workflow? (unfortunately I'm not able to do it myself)

File: .github/workflows/build-and-test.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ on:
   pull_request:
 
 env:
-  GO_VERSION: 1.19
+  GO_VERSION: 1.20
```
Comment (benkeith-splunk) [inline .github/workflows/build-and-test.yml]: Put quotes around the `"1.20"`. I think it is interpreting it incorrectly and trying to use Go 1.2 instead.

Comment (benkeith-splunk) [reply .github/workflows/build-and-test.yml]:
```suggestion
  GO_VERSION: "1.20"
```

Comment (dwalasek) [reply .github/workflows/build-and-test.yml]: Yes!!! That fixed the issue. Thank you for your help!

**Pull request merged**